### PR TITLE
chore: simplify cost calc

### PR DESCRIPTION
**Pull request by mattsse**
use existing tx functions

### CODE REVIEW

File: crates/transaction-pool/src/traits.rs [hunk 1]
```diff
@@ -1133,30 +1133,12 @@ impl EthPooledTransaction {
...
-                U256::from(t.gas_price).saturating_mul(U256::from(t.gas_limit))
-            }
-            Transaction::Eip2930(t) => {
-                U256::from(t.gas_price).saturating_mul(U256::from(t.gas_limit))
-            }
-            Transaction::Eip1559(t) => {
-                U256::from(t.max_fee_per_gas).saturating_mul(U256::from(t.gas_limit))
-            }
-            Transaction::Eip4844(t) => {
-                blob_sidecar = EthBlobTransactionSidecar::Missing;
```
Comment (klkvr) [inline crates/transaction-pool/src/traits.rs:1150]: this is now never set to `Missing`, should we still do it?

Comment (mattsse) [reply crates/transaction-pool/src/traits.rs:1150]: oh right

Comment (mattsse) [reply crates/transaction-pool/src/traits.rs:1150]: fixing

Comment (mattsse) [reply crates/transaction-pool/src/traits.rs:1150]: done

**Review by klkvr — APPROVED** [review]

**Pull request merged**